### DISCUSSION TITLE
'montage' centered by height

### DISCUSSION THREAD

**Discussion by murilo-kendjy** [ImageMagick/ImageMagick#4749] [Help] [answered]
I'm not sure if this is the best way to do it, but I'm using montage command to unity two figures: Image A (2580x1656) and Image B (526x2720), the problem is that the result is not centered aligned by height

The command I'm using:
`montage ImageA.png ImageB.png -tile 2x -geometry +15+0 result.png`

The result I get: (The graph on the left is Image A, and the long one in the right is Image B)

<a href="https://ibb.co/YQjmZzL"><img src="https://i.ibb.co/dMKFgd5/Res-energy.png" alt="Res-energy" border="0"></a>

The result I want is ImageA shifted down to be centered in height with ImageB.

**Comment by snibgo** [comment, marked as answer]
I don't use montage much, preferring to simply append images together, like this:
```
magick ImageA.png -size 15x1 xc:None ImageB.png -gravity Center +append +repage out.png
```

**Reply by murilo-kendjy** [reply]
Thanks, that worked.
